src/core/currency/wallet/currency-wallet-callbacks.js

```diff
@@ -189,6 +189,26 @@ export function makeCurrencyWalletCallbacks(
       for (const tx of txs) {
         const { txid } = tx
 
+        if (tx.confirmations == null) {
+          // Default confirmation rule is 1 block
+          const {
+            requiredConfirmations = 1
+          } = input.props.walletState.currencyInfo
+          const { height } = input.props.walletState
+          const blockConfirmations = height - tx.blockHeight
+
+          tx.confirmations =
+            height === 0
+              ? null
```

Comment:
Confirmations cannot be 'null'.
Look at the type and also the precommit tests fails.

Reply:
Fixed. I meant to keep it `undefined` if the block height for the wallet is 0 (not synchronized).